src/dfx/assets/new_project_node_files/webpack.config.js

```diff
@@ -86,10 +55,10 @@ module.exports = {
       template: path.join(__dirname, frontend_entry),
       cache: false,
     }),
-    new webpack.EnvironmentPlugin({
-      NODE_ENV: "development",
-      ...canisterEnvVariables,
-    }),
+    new webpack.EnvironmentPlugin([
+      ...Object.keys(process.env).filter((key) => key.includes("CANISTER_ID")),
```

Comment:
Are the env variables that contain `CANISTER_CANDID_PATH` in the name ever needed here?

Reply:
not typically

Reply:
That sounds like "we don't know."  In that case, please make this continue to provide the `CANISTER_CANDID_PATH` environment variables previously provided, in order to not introduce breaking changes

Reply:
all current dfx environment variables will be provided now